internal/generator/fluentd/fluent_conf_test.go

```diff
@@ -707,7 +707,7 @@ var _ = Describe("Generating fluentd config", func() {
   #remove structured field if present
   <filter **>
 	@type record_modifier
-    char_encoding ascii-8bit:utf-8
+    char_encoding utf-8:utf-8
```

Comment:
ascii-8bit is fluents default encoding which from my understanding is just "binary'.  The reason we introduced this encoding change was to address bad characters being shipped to Elasticsearch where the chars were invalid.  I don't believe this conversion is a no-op.  You could test by removing this char encoding all-to-gether

Reply:
I tested, and it works well, at least for my test case. 
You can take a look at this method https://github.com/repeatedly/fluent-plugin-record-modifier/blob/master/lib/fluent/plugin/filter_record_modifier.rb#L145. 
If i understand correct it will check is it valid utf8, if not replace.

Reply:
+1 the goal of the utf-8:utf-8 re-encoding is not to actually change the encoding, but to force the collector to decode the input as utf-8, so it can replace invalid sequences. Then its re-encoded as utf-8 again, but now with invalid sequences replaced.